### PR TITLE
add popup to display answer and check if correct

### PR DESCRIPTION
**Pull request by rbolline25**

### CODE REVIEW

File: src/question_view.py [hunk 1]
```diff
@@ -15,6 +15,10 @@ def create_question_view(tk, question_frame):
                               font=helvetica_20, bg=Color.LIGHT_GREEN.description, fg=Color.BLACK.description,
                               wraplength=350, justify='left')
     question_label.grid(row=0, column=0, sticky=tk.E)
+    
+    def showAnswer():
```
Comment (skasim) [inline src/question_view.py]: This new method be added outside of the `create_question_view_method()`

**Comment by skasim** [comment]
@rbolline25 Clicked on the Show Answer button and am able to see pop up

**Review by muftogun — APPROVED** [review]
I am now able to see answer popup - so looks good to me

File: src/question_view.py (hunk 1 above)
Comment (rbolline25) [reply src/question_view.py]: you're right, I've added it outside `create_question_view_method()`

**Pull request merged**